src/createServer.js

```diff
@@ -1,11 +1,164 @@
+
 'use strict';
 
-// const express = require('express');
+const express = require('express');
 
 function createServer() {
-  // Use express to create a server
-  // Add a routes to the server
-  // Return the server (express app)
+  const app = express();
+
+  let expenses = [];
+  let users = [];
+
+  app.use(express.json());
+
+  app.post('/users', (req, res) => {
+    const { name } = req.body;
+
+    if (!name) {
+      res.sendStatus(400);
+    }
+
+    const newUser = {
+      id: Math.floor(Math.random() * 1000),
+      name,
+    };
+
+    users.push(newUser);
+    res.statusCode = 201;
+    res.send(newUser);
```

Comment:
here use use `res.statusCode + res.send` in code below `(line 49-50) `res.sendStatus + res.send` in kind of the same case(when you use `send` not only status send). Better do with the same logic all over the code. Firs option - `res.statusCode + res.send` as for me - preferable.